tests/websocket.c

```diff
@@ -8,319 +8,537 @@
 #include <pthread.h>
 #include <arpa/inet.h>
 #include <errno.h>
+#include <ctype.h>
+#include <getopt.h>
 
 #include <sys/types.h>
 #include <sys/socket.h>
 
 #include <event.h>
+#include <http_parser.h>
+
+#define DEBUG_LOGS 0
 
 struct host_info {
-	char *host;
-	short port;
+    char *host;
+    short port;
```

Comment:
hmm… the whole repo uses tabs. I do prefer spaces but I'm not sure this is the right place to start changing the indentation.